src/state/slices/opportunitiesSlice/resolvers/idle/types.ts

```diff
@@ -0,0 +1,5 @@
+import type { OpportunityMetadataBase } from '../../types'
+
+export type IdleStakingSpecificMetadata = OpportunityMetadataBase & {
+  cdoAddress: string
```

Comment:
A comment on what a `cdoAddress` is would be useful here.

Reply:
05d16675cbf